control-plane/p4RuntimeArchHandler.h

```diff
@@ -209,7 +214,7 @@ bool isExternPropertyConstructedInPlace(const IR::P4Table* table,
 /// Visit evaluated blocks under the provided top-level block. Guarantees that
 /// each block is visited only once, even if multiple paths to reach it exist.
 template <typename Func>
-void forAllEvaluatedBlocks(const IR::ToplevelBlock* aToplevelBlock, Func function) {
+void forAllEvaluatedBlocks(const IR::Block* aToplevelBlock, Func function) {
```

Comment:
Is this comment and parameter name incorrect?

Reply:
thanks, updated to make it generic